recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ requirements:
 
   run:
     - python
+    - networkx
     - colormath
```

Comment:
This needs
```yaml
  - networkx ==2.0
  - colormath ==3.0.0
```
due to
https://github.com/jsvine/spectra/blob/v0.0.10/requirements.txt
https://github.com/jsvine/spectra/blob/v0.0.10/setup.py#L2
https://github.com/jsvine/spectra/blob/v0.0.10/setup.py#L9-L10
https://github.com/jsvine/spectra/blob/v0.0.10/setup.py#L40